htdocs/.gitignore

```diff
@@ -0,0 +1,16 @@
+/app/config/parameters.yml
```

Comment:
We already have .gitignore https://github.com/GrossumUA/Symfony3-Base-Instance/blob/master/.gitignore